base_layer/wallet/src/transaction_service/handle.rs

```diff
@@ -378,10 +378,10 @@ impl fmt::Display for TransactionEvent {
                 write!(f, "Transaction validation state changed: {}", operation_id)
             },
             TransactionEvent::TransactionValidationCompleted(operation_id) => {
-                write!(f, "Transaction validation completed: {}", operation_id)
+                write!(f, "Transaction validation(#{}) completed", operation_id)
```

Comment:
You know that you can do this now, right? Great for readability.
```suggestion
                write!(f, "Transaction validation(#{operation_id}) completed")
```

Reply:
No I did not, but that's great